server/container/guice/cassandra-rabbitmq-guice/src/test/resources/rabbitmq.properties

```diff
@@ -0,0 +1,2 @@
+uri=amqp://james:james@rabbitmq_host:5672
```

Comment:
This file is, as far as I am aware of, overwritten.

If never used please remove it...